From beab5dfdac5e1ec39ac7ee4cdd127e3f12107e67 Mon Sep 17 00:00:00 2001
From: Max Ihlenfeldt <max@igalia.com>
Date: Tue, 11 Apr 2023 15:36:16 +0000
Subject: [PATCH] Backport "Fix deps for feed_response.proto"

This is a backport of a7d41a87 (crrev.com/c/4328228) to fix a build race
condition (a dependency wasn't declared in GN, which may lead to a build
failure if the dependency is built too late).

This patch can be dropped when we update to Chromium 113.

Upstream-Status: Inappropriate
Signed-off-by: Max Ihlenfeldt <max@igalia.com>
---
 components/feed/core/proto/BUILD.gn | 1 +
 1 file changed, 1 insertion(+)

diff --git a/components/feed/core/proto/BUILD.gn b/components/feed/core/proto/BUILD.gn
index f0ec5318afb07..45b8a786de973 100644
--- a/components/feed/core/proto/BUILD.gn
+++ b/components/feed/core/proto/BUILD.gn
@@ -66,6 +66,7 @@ proto_library("proto_v2") {
     "v2/wire/xsurface_content.proto",
     "v2/xsurface.proto",
   ]
+  proto_deps = [ ":proto_ios_shared_v2" ]
 }

 proto_library("proto_ios_shared_v2") {
